examples/standalone/data-policy.yaml

```diff
@@ -65,6 +65,23 @@ rule_sets:
           - principals: [ ]
             fixed:
               value: "****"
+      - field:
+          name_parts: [ transactionamount ]
+        transforms:
+          - principals: [ { group: administrator } ]
+            identity: { }
+          - principals: [ { group: marketing } ]
+            aggregation:
+              avg:
+                cast_to: float4
+          - principals: [ { group: fraud_and_risk} ]
+            aggregation:
+              avg: {
+                precision: 1
+              }
+          - principals: [ ]
+            aggregation:
+              sum: {}
```

Comment:
Weren't we going to remove these from the standalone examples?

Reply:
good catch